homeassistant/components/binary_sensor/nest.py

```diff
@@ -8,9 +8,9 @@
 import logging
 
 from homeassistant.components.binary_sensor import (BinarySensorDevice)
+from homeassistant.components.nest import DATA_NEST, EVENT_NEST_UPDATE
 from homeassistant.components.sensor.nest import NestSensor
```

Comment:
It's not allowed to depend on another platform from a platform. You can define a common base nest entity class instead in the nest component and import that into the nest platforms.

Reply:
That is old code. Do you want to me fix that in this PR?

Reply:
It can be done separately. I mentioned it since the event handler that is defined in the sensor nest base class is now used in this binary sensor platform.

Reply:
Will do that later